eventmesh-common/src/main/java/org/apache/eventmesh/common/ThreadPoolFactory.java

```diff
@@ -35,26 +35,25 @@ public static ThreadPoolExecutor createThreadPoolExecutor(int core, int max, fin
     }
 
     public static ThreadPoolExecutor createThreadPoolExecutor(int core, int max, final String threadName,
-        final boolean isDaemon) {
+                                                              final boolean isDaemon) {
         return createThreadPoolExecutor(core, max, new LinkedBlockingQueue<>(1000), threadName, isDaemon);
     }
 
     public static ThreadPoolExecutor createThreadPoolExecutor(int core, int max, BlockingQueue<Runnable> blockingQueue,
-        final String threadName, final boolean isDaemon) {
+                                                              final String threadName, final boolean isDaemon) {
         return new ThreadPoolExecutor(core, max, 10 * 1000, TimeUnit.MILLISECONDS, blockingQueue,
```

Comment:
I am afraid that aligning the parameters of the second line forcibly with the starting point of the first line parameters may result in all the parameters being piled up on the right side, especially when the method name is long. This requires the community for their opinions.

Furthermore, when coding with IDEA, IDE will align the parameters of the second line with the method name. Developers may place multiple parameters on the second line, which can be passed by the CheckStyle plugin. However, when the code is committed, Spotless will shift the parameters of the second line to the right, causing the second line to exceed the length limit and consequently causing the CheckStyle CI check to fail.

Reply:
I agree with, I don't think it is necessary to align according to the first parameter of the method first, and then turn this alignment strategy on when the community thinks it needs to be aligned.